index.js

```diff
@@ -33,9 +39,9 @@ module.exports = function(glob, options) {
 
   // make glob absolute
   if (rootDir && glob.charAt(0) === '/') {
-    glob = path.join(rootDir, glob);
-  } else if (!isAbsolute(glob)) {
-    glob = path.resolve(cwd, glob);
+    glob = join(rootDir, glob);
+  } else if (process.platform === 'win32' || !isAbsolute(glob)) {
```

Comment:
See my change at https://github.com/phated/to-absolute-glob/commit/cd17408426a75dcebd930f02a41c13d8b5de8c20 which caused a passing test suite at https://ci.appveyor.com/project/gulpjs/glob-stream/build/1.0.54